addons/website/static/src/builder/plugins/form/form_option.xml

```diff
@@ -88,6 +91,9 @@
     <t t-if="domState.elClassList.includes('s_website_form_model_required')">
         <FormModelRequiredFieldAlert fieldName="domState.fieldName" modelName="domState.modelName" fetchModels="props.fetchModels"/>
     </t>
+    <t t-elif="domState.elClassList.includes('s_send_a_copy_field')">
+        <FormFieldAlert message='`The field "${ domState.fieldName }" is mandatory when the option "Send a Copy" is enabled`'/>
+    </t>
```

Comment:
Maybe I misunderstand what this is about.

I did the following, with `website_sale` installed:
- set form actions as "Create a Customer"
- remove e-mail field
- enable "Send a copy"

=> No such warning

- set form actions as "Create a Customer"
- enable "Send a copy"
- remove e-mail field

=> No such warning